rootfs/api/models/app.py

```diff
@@ -211,6 +211,17 @@ def create(self, *args, **kwargs):  # noqa
             except KubeException:
                 self._scheduler.ns.create(namespace)
 
+            if settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC != '':
+                quota_name = '{}-quota'.format(namespace)
+                self.log(settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC)
+                quota_spec = json.loads(settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC)
+                self.log('creating Quota {} for namespace {}'.format(quota_name, namespace),
+                         level=logging.DEBUG)
+                try:
+                    self._scheduler.quota.get(namespace, quota_name)
```

Comment:
It's a bad idea, because an administrator may already changes quota through kubernetes interface for some apps and restart of controller rollback this changes.  But yes, there is problem.